docs/organizations/roles-and-permissions-for-organizations.md

```diff
@@ -426,6 +426,7 @@ To change this, open your organization **Settings**, page **Roles and permission
 -   [Configure code patterns](../repositories-configure/configuring-code-patterns.md)
 -   [Configure file extensions](../repositories-configure/file-extensions.md)
 -   [Manage branches](../repositories-configure/managing-branches.md)
+-   [Reanalyze branches and pull requests](../faq/repositories/how-do-i-reanalyze-my-repository.md)
 
 ![Configuring who can change analysis configurations](images/roles-permissions-repo-management.png)
```

Comment:
@claudiacodacy @joanasteodoro 
Under the scope of [PLUTO-587](https://codacy.atlassian.net/browse/PLUTO-587), the permission level set for analysis configuration now includes one more operation: the **reanalysis of branches and pull requests**. This behavior is already in production.

This pull request updates the documentation for this change, but we must also reflect this change on the UI. 🙏 

![image](https://github.com/codacy/docs/assets/26439949/e5502684-62f6-4c33-80d5-d005a13bc866)

I would propose something like this:
**Define which roles can configure patterns, configure analyzed file extensions and branches, ignore issues and files, and reanalyze branches and pull requests.**

Note: I'm leaving this note here because I believe you are still tweaking the UI. 😉 

CC @nicklem